wallet/src/Pos/Wallet/Web/Methods/History.hs

```diff
@@ -215,3 +230,11 @@ logTxHistory desc = do
         . sformat (stext%" transactions history: "%listJson) desc
         . map _thTxId
         . toList
+
+logCTxs
+    :: (Container t, Element t ~ CTx, WithLogger m)
+    => Text -> t -> m ()
+logCTxs desc =
```

Comment:
Maybe we shouldn't stop the PR from being merged, but I think these kind of functions can backfire really rapidly if we do not limit the number of elements logged _internally_, rather than externally.

Reply:
It's a valid concern, but not for this PR, because the only goal here is to merge `master` to `develop`.